src/cluster/src/install.rs

```diff
@@ -844,11 +863,55 @@ impl ClusterInstaller {
         Ok(None)
     }
 
+    /// Wait until the Fluvio SC public service appears in Kubernetes
+    async fn wait_for_sc_port_check(&self, addr: &str) -> Result<Option<String>, ClusterError> {
+        for i in 0..30u16 {
+            info!(attempt = i, "waiting for sc port check");
+            let sock_addr_string = format!("{}:9003", addr);
```

Comment:
can't assume 9003 port, should use whatever configuration in the cluster installer

Reply:
That's what it was doing before. I'll update to use config value